easybuild/easyconfigs/n/Normaliz/Normaliz-3.6.3-intel-2018b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'Normaliz'
+version = '3.6.3'
+
+homepage = 'https://www.normaliz.uni-osnabrueck.de/'
+description = """Normaliz is an open source tool for computations in affine monoids,
+vector configurations, lattice polytopes, and rational cones."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/Normaliz/Normaliz/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['701b39189758eb9a13e189ea1a43218e1c001f19c228e761a80949d610fe3900']
+
+dependencies = [('GMP', '6.1.2')]
+
+sanity_check_paths = {
+    'files': ['bin/normaliz', 'bin/Qnormaliz', 'lib/libnormaliz.so', 'lib/libQnormaliz.so'],
```

Comment:
@wpoely86 Please use `SHLIB_EXT` rather than hardcoding `.so`

```python
sanity_check_paths = {
    'files': ['bin/normaliz', 'bin/Qnormaliz', 'lib/libnormaliz.%s' % SHLIB_EXT, 'lib/libQnormaliz.%s' % SHLIB_EXT],
    'dirs': ['include/libnormaliz']
}